### PR TITLE
[ENH] Standardize and organize entity descriptions

### PR DESCRIPTION
**Pull request by tsalo**
Closes None. I was starting to work on #612 and #1194, when I realized that the actual entity definitions could be improved.
My main goal was to make the link between entities and metadata fields clearer.

### CODE REVIEW

File: src/schema/objects/entities.yaml [hunk 1]
```diff
@@ -290,40 +308,37 @@ space:
   name: space
   display_name: Space
   description: |
-    The space entity can be used to indicate
-    the way in which electrode positions are interpreted
-    (for EEG/MEG/iEEG data) or
-    the spatial reference to which a file has been aligned (for MRI data).
-    The space `<label>` MUST be taken from one of the modality specific lists in
+    The `space-<label>`` entity can be used to indicate the way in which electrode positions are interpreted
```
Comment (sappelhoff) [inline src/schema/objects/entities.yaml:311]:
```suggestion
    The `space-<label>` entity can be used to indicate the way in which electrode positions are interpreted
```

File: src/schema/objects/entities.yaml [hunk 2]
```diff
@@ -290,40 +308,37 @@ space:
...
   type: string
   format: label
 split:
   name: split
   display_name: Split
   description: |
-    In the case of long data recordings that exceed a file size of 2Gb, the
-    .fif files are conventionally split into multiple parts.
+    In the case of long data recordings that exceed a file size of 2Gb,
+    `.fif`` files are conventionally split into multiple parts.
```
Comment (sappelhoff) [inline src/schema/objects/entities.yaml:329]:
```suggestion
    `.fif` files are conventionally split into multiple parts.
```

File: src/schema/objects/entities.yaml [hunk 3]
```diff
@@ -290,40 +308,37 @@ space:
...
 
     Instead of a simple renaming, files should be read in and saved under their
     new names with dedicated tools like [MNE-Python](https://mne.tools/),
-    which will ensure that not only the file names, but also the internal file
-    pointers will be updated.
-    It is RECOMMENDED that .fif files with multiple parts use the
-    `split-<index>` entity to indicate each part.
+    which will ensure that not only the file names, but also the internal file pointers, will be updated.
+
+    It is RECOMMENDED that .fif files with multiple parts use the `split-<index>` entity to indicate each part.
```
Comment (sappelhoff) [inline src/schema/objects/entities.yaml:337]:
```suggestion
    It is RECOMMENDED that `.fif` files with multiple parts use the `split-<index>` entity to indicate each part.
```

File: src/schema/objects/entities.yaml (hunk 1 above)
Comment (tsalo) [reply src/schema/objects/entities.yaml]: Thanks for catching these @sappelhoff

**Comment by sappelhoff** [comment]
thanks @tsalo

**Pull request merged**